src/Symfony/Component/PasswordHasher/Hasher/UserPasswordHasherInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PasswordHasher\Hasher;
+
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * Interface for the user password hasher service.
+ *
+ * @author Ariel Ferrandini <arielferrandini@gmail.com>
+ */
+interface UserPasswordHasherInterface
+{
+    /**
+     * Hashes the plain password.
+     */
+    public function hashPassword(UserInterface $user, string $plainPassword): string;
+
+    public function isPasswordValid(UserInterface $user, string $raw): bool;
```

Comment:
```suggestion
    public function isPasswordValid(UserInterface $user, string $plainPassword): bool;
```

I think it's good to use only "plaintext"/"plain password" to be clear on what is expected.

Reply:
Maybe also add a PHPdoc here, e.g: "Checks if the plaintext password matches the user's password"